platform/compound/client.go

```diff
@@ -0,0 +1,30 @@
+package compound
```

Comment:
compund is not a blockchain - why it is different platform?

Reply:
I didn't want to put into ethereum, not to tie it too close to eth transfer stuff.
I didn't want to put outside of platform yet.
The current platform structure is outdated, from the times there were only TX support.  Some parts of atlas don't fit to it, like name lookup.  I want to rearrange the structure, not to force non-tx, non-strictly-chain-related parts into the platform structure, but I did not want to start the change with lending.
Maybe that would be the right approach, though.  Keep it in separate lending folder.

Reply:
Moved compound to under ethereum.